packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -224,19 +238,29 @@ const ReviewVoteTableRow = (
     positiveVoteCountTooltip = "2 or more positive votes"
   }
 
+  const userReviewVote = 
+    post.currentUserReviewVote?.quadraticScore 
+    || (post.currentUserReviewVote?.qualitativeScore ? getCostData({costTotal})[post.currentUserReviewVote?.qualitativeScore].value : "")
```

Comment:
Would be good to split this up into a few lines for readability.  If `quadraticScore` is 0, do we want to coalesce it (which `||` will do), or do we only want to do that if it's missing entirely?  In which case use `??`.

I'd probably split this up like this:
```
const qualitativeScore = post.currentUserReviewVote?.qualitativeScore;
const qualitativeScoreDisplay = qualitativeScore ? getCostData({costTotal})[qualitativeScore].value : "";
const userReviewVote = post.currentUserReviewVote?.quadraticScore ?? qualitativeScoreDisplay;
```